openwisp_radius/migrations/0002_initial_openwisp_radius.py

```diff
@@ -0,0 +1,241 @@
+# Generated by Django 2.1.1(2018-09-27) + Manually Edited
+
+import re
+import uuid
+
+import django.core.validators
+import django.db.models.deletion
+import django.utils.timezone
+import model_utils.fields
+from django.conf import settings
+from django.db import migrations, models
+from django_freeradius.migrations import add_default_group_to_existing_users, add_default_groups
+from openwisp_radius.migrations import add_default_organization
+
+import openwisp_radius.models
+import openwisp_users.mixins
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('openwisp_users', '0003_default_organization'),
```

Comment:
Yes,it's the 0003 migration of **openwisp_users**.
1. It was automatically generated.
2. It is required because before running this file i want the default_organisation set from the openwisp_users migrations.

Reply:
oh right, I'm so sorry I was quick to read that as `openwisp_radius`, you should install the master of openwisp-users in `.travis.yml` then, [as we are doing for django-freeradius](https://github.com/openwisp/openwisp-radius/blob/master/.travis.yml#L16-L17)